commons-model/src/main/java/com/appjars/saturn/model/BaseEntity.java

```diff
@@ -64,5 +65,14 @@ public boolean equals(Object obj) {
 		}
 		return true;
 	}
+	
+	@Override
+	public String toString() {
+		if (id!=null) {
+			return this.getClass().getName()+"@"+id;
+		} else {
+			return this.getClass().getName()+"@transient;"+Integer.toHexString(System.identityHashCode(this));
```

Comment:
Maybe something silly, but why semicolon? maybe with colon instead it's nicer:
`com.test.model.MyPojo@transient:2101973421`
vs
`com.test.model.MyPojo@transient;2101973421`

Reply:
Done ✔